IDE/Espressif/ESP-IDF/examples/wolfssl_benchmark/main/main.c

```diff
@@ -98,22 +114,26 @@ void my_atmel_free(int slotId)
 #endif /* CUSTOM_SLOT_ALLOCATION                                       */
 #endif /* WOLFSSL_ESPWROOM32SE && HAVE_PK_CALLBACK && WOLFSSL_ATECC508A */
 
+/* the following are needed by benchmark.c with args */
+#ifdef WOLFSSL_BENCH_ARGV
+char* __argv[22];
```

Comment:
Why 22, if an arbitrary current max then please use a macro. Something like MAX_ARGUMENTS and check it against cnt in the loop below.

Reply:
Good catch! I've fixed the argument count in [this commit](https://github.com/wolfSSL/wolfssl/pull/5800/commits/8a4193b1c2637b728a723e6ccafcae461ddeaf68) and added a comment that the max value of 22 is arbitrary and now defined in `WOLFSSL_BENCH_ARGV_MAX_ARGUMENTS`.